pkg/pillar/cmd/downloader/status.go

```diff
@@ -18,7 +18,9 @@ type PublishStatus struct {
 }
 
 // Progress report progress as a percentage of completeness
-func (d *PublishStatus) Progress(p uint) {
+func (d *PublishStatus) Progress(p uint, osize, asize int64) {
```

Comment:
Not your name, but I never figured out how "o" could mean current and "a" total. I wouldn't mind if the variables and parameters where renamed at some point in time (or someone else can clue me in on the names.)

Reply:
Neither could I. I checked it multiple times in this brief PR to make sure I wasn't messing something up. I am more than supportive of a fix in a separate PR.